pipelines/src/main/groovy/BuildEnvStub.groovy

```diff
@@ -0,0 +1,9 @@
+abstract class BuildEnvStub {
```

Comment:
Worth adding a comment somewhere in this project about why you're using this approach